src/components/EpisodeCard/EpisodeCard.js

```diff
@@ -34,12 +41,32 @@ export default function EpisodeCard({ episode }) {
           __html: DOMpurify.sanitize(episode.description),
         }}
       />
-      <Audio controls={true} src={episode.enclosureUrl}></Audio>
-      <Span>{`${dateFormat} ${
-        duration ? `● ${time === 0 ? 1 : time} min.` : ''
-      }`}</Span>
+
+      <BottomContainer>
+        <Span>{`${dateFormat} ${
+          duration ? `● ${time === 0 ? 1 : time} min.` : ''
+        }`}</Span>
+        <PlayButton onClick={handleClick}>
```

Comment:
Maybe be a bit more precise with your naming. Name the function after what it does. `handleClick` is very generic!

Reply:
Implemented a better name.